core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1611,6 +1668,95 @@ private void validateTableModelAndCreateTypeCast(
             throw SqlException.position(0).put("timestamp is not defined");
         }
     }
+    
+    private CompiledQuery sqlBackup(SqlExecutionContext executionContext) throws SqlException {
+		final CharSequence tok = SqlUtil.fetchNext(lexer);
+		if (null != tok) {
+			if (Chars.equalsLowerCaseAscii(tok, "table")) {
+				return sqlTableBackup(executionContext);
+			}
+		}
+		throw SqlException.position(lexer.getPosition()).put(" expected 'table'");
+      }
+    
+	private CompiledQuery sqlTableBackup(SqlExecutionContext executionContext) throws SqlException {
+		ObjList<CharSequence> tableNames = new ObjList<>();
```

Comment:
This perhaps need to be a set? What if someone specifies the same table name several times in the list of tables?

This needs a test too.